components/autogen/src/format-pages.txt

```diff
@@ -1222,10 +1222,10 @@ mif = true
 pagename = leica-lif
 extensions = .lif
 owner = `Leica <http://www.leica.com/>`_
-developer = `Leica Microsystems CMS GmbH <http://www.leica-microsystems.com/>`_
+developer = `Leica Microsystems CMS GmbH <https://www.leica-microsystems.com/>`_
 bsd = no
 versions = 1.0, 2.0
-software = `Leica LAS AF Lite <http://www.leica-microsystems.com/products/microscope-software/software-for-life-science-research/las-x/>`_ (links at bottom of page)
+software = `Leica LAS AF Lite <https://www.leica-microsystems.com/products/microscope-software/software-for-life-science-research/las-x/>`_ (links at bottom of page)
```

Comment:
Being strictly accurate, the download links for now just over half way down the page rather than at the bottom - change this text to "(scroll down to the Downloads section)"?

Reply:
Or 'see Downloads section'?